docs/learn/learn-staking.md

```diff
@@ -9,189 +9,355 @@ slug: ../learn-staking
+held accountable for their bad actions. In NPoS, validators are paid equal rewards regardless of the amount of stake backing them, thus avoiding large payouts to few large validators which might lead to centralization.
+
+## Being a nominator
+
+### Tasks and Responsibilities of a Nominator
+
+Since validator slots are limited, most of those who wish to stake their DOT and contribute to the
+economic security of the network will be nominators, thus here we focus on the role of nominators.
+However, it is worth mentioning that validators do most of the heavy lifting: they run the validator nodes and manage
+[session keys](https://research.web3.foundation/en/latest/polkadot/keys/index.html?highlight=session%20keys),
+produce new block candidates in [BABE](learn-consensus.md/#block-production-babe), vote and come to
+consensus in [GRANDPA](learn-consensus.md/#finality-gadget-grandpa), validate the state transition
+function of parachains, and possibly some other responsibilities regarding data availability and
+[XCM](learn-cross-consensus.md). For more information, you can take a look at the
+[validator docs](learn-validator.md) to understand what you need to do as a validator. If you want
+to become a validator you can consult [this](../maintain/maintain-guides-how-to-validate-polkadot.md) guide.
+
+Nominators have far fewer responsibilities than validators. These include selecting validators and
+monitoring their performance, keeping an eye on changing commission rates (a validator can change
+commission at any time), and general health monitoring of their validators' accounts. Thus, while not
+being completely set-it-and-forget-it, a nominator's experience is relatively hands-off compared to that of
+a validator. For more information, you can take a look at the [nominator guide](learn-nominator.md)
+to understand your responsibilities as a nominator.
+
+If you want to become a nominator, see
+[this](../maintain/maintain-guides-how-to-nominate-polkadot.md) guide. If you are a beginner and
+would like to securely stake your tokens using Polkadot JS Apps, refer to [this](https://support.polkadot.network/support/solutions/articles/65000168057-how-do-i-stake-nominate-on-polkadot-) support article. {{ kusama: The tutorial presented in the support article is demonstrated on Polkadot, but the procedure is the same for Kusama :kusama }}
+
+### Selection of Validators
+
+The task of choosing validators is not simple, as it should take into account nominator reward and
+risk preferences. Ideally one aims to maximize the reward-to-risk ratio by maximizing rewards and minimizing risks, with sometimes having to
+compromise between the two, as minimizing risks might decrease rewards as well. Nominators should pay attention, especially to six criteria when nominating validators (not in order of importance):
+
+- recent history of the era points earned across eras
+- validator's (own) stake (shows skin in the game)
```

Comment:
We refer to Validator's own stake as Self Stake https://wiki.polkadot.network/docs/maintain-guides-validator-community#self-stake You could also link to this doc here

Reply:
Alright then I need to change the figure. I took the terminology from the staking tab on the UI.

Reply:
Got it. In that case, let us leave it as is. Should we may be add a snapshot of Polkadot JS UI along with the figure and pointing to the respective parameters?

Reply:
New Figures added, updated Figure risk-rewards 👍